xonsh/environ.py

```diff
@@ -585,6 +585,9 @@ def swap(self, other):
     #
 
     def __getitem__(self, key):
+        if key is Ellipsis:
+            val = builtins.__xonsh_env__
+            return val
```

Comment:
I think it would be better to just return `self` in the unfortunate event that someone overwrites `builtins.__xonsh_env__` but keeps a reference to it elsewhere.